react/src/pages/user/profile/ProfilePage.jsx

```
@@ -24,32 +24,61 @@ const ProfilePage = () => {
+        </div>
+      </div>
+    );
   }
 
   if (!user) {
-    return null;
+    return (
+      <div className={styles.profileContainer}>
+        <div className={styles.error}>
+          <p>{t('profile.noUser')}</p>
+          <button onClick={() => navigate('/login')}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)